src/app/frontend/filters_info.py

```diff
@@ -0,0 +1,28 @@
+import streamlit as st
+
+EMPTY_FILTERS = "No filters applied"
+NON_EMPTY_FILTERS = "Filters applied -\n\n"
+
+
+def show_filters_info(applied_filters: dict) -> None:
```

Comment:
Can you specify further here about the typehint for `dict`? is it `Dict[str, Any]` or specifically this is just a combo of `Dict[str, str]` only?

Reply:
Yes, this makes sense! It will be `Dict[str, Any]`